### PR TITLE
[WIP] Detect if this is a Raspbian Jessie install

### PR DESCRIPTION
**Pull request by dschaper**
Signed-off-by: Dan Schaper <dan.schaper@pi-hole.net>

**By submitting this pull request, I confirm the following (please check boxes, eg [X]) _Failure to fill the template will close your PR_:**

***Please submit all pull requests against the `development` branch. Failure to do so will delay or deny your request***

- [x] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md).
- [x] I have checked that [another pull request](https://github.com/pi-hole/pi-hole/pulls) for this purpose does not exist.
- [x] I have considered, and confirmed that this submission will be valuable to others.
- [x] I accept that this submission may not be used, and the pull request closed at the will of the maintainer.
- [x] I give this submission freely, and claim no ownership to its content.

**How familiar are you with the codebase?:**

10

---

Only depend on `dhcpcd` if it's a real Raspbian Jessie installation.

_This template was created based on the work of [`udemy-dl`](https://github.com/nishad/udemy-dl/blob/master/LICENSE)._

### CODE REVIEW

File: automated install/basic-install.sh [hunk 1]
```diff
@@ -98,6 +98,12 @@ if command -v apt-get &> /dev/null; then
     return 0
   }
 
+  source /etc/os-release || true
+    if [[ ${PRETTY_NAME} == "Raspbian GNU/Linux 8 (jessie)" ]]; then
```
Comment (DL6ER) [inline automated install/basic-install.sh]: Will this finally break support for `wheezy`?

And what about `stretch`? Will it still be using `dhcpcd5`?

Comment (dschaper) [inline automated install/basic-install.sh]: Wheezy uses `/etc/network/interfaces` and I'm betting that `stretch` will keep `dhcpcd`. The RPF (Raspbian) uses that as an assistant for their GUI configuration of Raspbian, and I don't see them changing Pixel now. 

The other fun fact is that they have released Raspbian Jessie for x86 (Live environment only I believe, I don't think you can use it as a full desktop replacement yet.) Haven't the motivation to boot that thing on any hardware I have here to check it out.

Comment (dschaper) [inline automated install/basic-install.sh]: I'm almost inclined to just leave that out of the dependencies now, anyone that is installing Raspbian Jessie would already have the package installed, we don't need to add it anymore. Then the `/etc/dhcpcd.conf` file detection would be enough to check if that file needed modification.

**Comment by dschaper** [comment]
Fixes #830

**Comment by dschaper** [comment]
This is turning into a total rewrite of the IPv4 configuration detection and correction of the problems with multiple entries in the `/etc/dhcpcd.conf` file that we've put in there. Might just be something for v3 and just overhaul the whole process.

**Comment by jacobsalmela** [comment]
This should fix #830. Just posting this here for documentation.

**Comment by unclehowell** [comment]
Guys, is there a solution for auto selecting the defaults with the whiptail menu? I want to just deploy this beauty, it works, I trust it, why make users go through manual install

**Comment by PromoFaux** [comment]
@unclehowell our, [Discourse Forums](https://discourse.pi-hole.net), or [Subreddit](https://reddit.com/r/pihole) are a better place for discussion than unrelated issue threads

**Comment by dschaper** [comment]
This should be covered under the new installer plans. I don't see a need to keep this open.

**Pull request closed without merging**